sycl/include/sycl/reduction.hpp

```diff
@@ -140,12 +140,19 @@ template <typename... Ts> ReduTupleT<Ts...> makeReduTupleT(Ts... Elements) {
   return sycl::detail::make_tuple(Elements...);
 }
 
+#if __INTEL_PREVIEW_BREAKING_CHANGES
```

Comment:
```suggestion
#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
```
Please, use unified style for this check.